packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -245,19 +279,21 @@ export const useArbTokenBridge = (
   async function withdrawEth({
     amount,
     l2Signer,
-    txLifecycle
+    txLifecycle,
+    destinationAddress
   }: {
     amount: BigNumber
     l2Signer: Signer
     txLifecycle?: L2ContractCallTransactionLifecycle
+    destinationAddress?: string
   }) {
     try {
       const ethBridger = await EthBridger.fromProvider(l2.provider)
 
       const tx = await ethBridger.withdraw({
         amount,
         l2Signer,
-        destinationAddress: walletAddress,
+        destinationAddress: destinationAddress || walletAddress,
```

Comment:
`destinationAddress ?? walletAddress`, right?

Reply:
Shouldn't get to the point where an empty address is passed, but if it is I'd prefer to default to the `walletAddress`